Source/WebCore/platform/ScrollableArea.cpp

```diff
@@ -68,15 +71,20 @@ ScrollAnimator& ScrollableArea::scrollAnimator() const
     return *m_scrollAnimator.get();
 }
 
-ScrollbarsController& ScrollableArea::scrollbarsController() const
+ScrollbarsController& ScrollableArea::scrollbarsController(Page* page) const
 {
     if (!m_scrollbarsController) {
         if (mockScrollbarsControllerEnabled()) {
             m_scrollbarsController = makeUnique<ScrollbarsControllerMock>(const_cast<ScrollableArea&>(*this), [this](const String& message) {
                 logMockScrollbarsControllerMessage(message);
             });
-        } else
-            m_scrollbarsController = ScrollbarsController::create(const_cast<ScrollableArea&>(*this));
+        } else {
+            if (page->chrome().client().isUsingUISideCompositing()) {
```

Comment:
If page can be null, we need to null check page here.

Reply:
Fixed.